README-Engine.md

```diff
@@ -1,308 +1,308 @@
 ENGINES
 =======
```

Comment:
This should probably have an early note saying the this is a deprecated feature that's replaced with providers, and that it's planned to go away.

Reply:
Good point. Please take a look at [0067c99cc592](https://github.com/openssl/openssl/pull/14042/commits/0067c99cc5923ab4c4e7512fdc8fd49410cda954) for the result.